Formula/c/chainhook.rb

```diff
@@ -0,0 +1,20 @@
+class Chainhook < Formula
+  desc "Reorg-aware indexing engine for the Stacks & Bitcoin blockchains"
+  homepage "https://github.com/hirosystems/chainhook"
+  url "https://github.com/hirosystems/chainhook.git",
+      tag:      "v1.4.1",
+      revision: "9d9f0d66dc32c7680fdabd3dba2ecf2847dd2b2a"
```

Comment:
please use tarball

Reply:
Hi, thanks for the feedback @chenrui333. 

Can I ask why we need to use a tarball? I used another comparable rust formula as a template (https://github.com/Homebrew/homebrew-core/blob/master/Formula/c/clarinet.rb), and it builds from a git revision as well.